_posts/2022-08-04-decade-innovation.md

```diff
@@ -0,0 +1,180 @@
+types](https://trino.io/docs/current/security/authentication-types.html)
+starting with LDAP and password file, to the modern OAuth2.0 and SSO systems.
+Trino supports fine-grained access control and [security management SQL commands
+like `GRANT` and
+`REVOKE`](https://trino.io/docs/current/language/sql-support.html#security-operations).
+You can secure connections from client tools, and use numerous methods to ensure
+secured access to your data sources.
+
+## Client tools and integrations
+
+In the very beginning all you could do is submit a query to the [client REST
+API](https://trino.io/docs/current/develop/client-protocol.html). Very quickly
+we added the [Trino CLI](https://trino.io/docs/current/installation/cli.html)
+and the [JDBC driver](https://trino.io/docs/current/installation/jdbc.html). And
+while it has continued to be widely used in the community, and gathered great
+features such as command-completion and history, different output formats, and
+much more, the Trino CLI is not the only tool anymore. The JDBC driver, the
+[Python client](https://github.com/trinodb/trino-python-client), the [Go
+client](https://github.com/trinodb/trino-go-client), and the ODBC driver from
+[Starburst](https://starburst.io/), all expanded the support for different
+client tools. You can query Trino in your Java-based IDE, such as IntelliJ
+IDEA, or database tool, such as [DBeaver](https://dbeaver.io/) or
+[Metabase](https://www.metabase.com/). You can take advantage of visualizations
+in [Apache Superset](https://superset.apache.org/), or automate with [Apache
+Airflow](https://airflow.apache.org/), [dbt](https://www.getdbt.com/), or
+[Apache Flink](https://flink.apache.org/). And many commercial tools such as
+[Tableau](https://www.tableau.com/), [Looker](https://www.looker.com/),
+[PowerBI](https://powerbi.microsoft.com/), or
+[ThoughtSpot](https://www.thoughtspot.com/) also proudly support Trino users.
+
+## SQL
+
+All the client tools and integrations rely on the rich SQL support of Trino,
+which has grown tremendously. Purely analytics-related support for `SELECT` and
+all its complexities was not enough. Trino gained support for data management to
+create schema and tables, but also views and materialized views. And with that
+[write support we needed `INSERT`, `UPDATE`, and
+`DELETE`](https://trino.io/docs/current/language/sql-support.html#write-operations).
+That’s all done and `MERGE` is next. But the core language features were not
+able to satisfy the needs of our users. We added functions for a large variety
+of topics ranging from simple string and [date
+functions](https://trino.io/docs/current/functions/datetime.html) to [JSON
+support](https://trino.io/docs/current/functions/json.html), [geospatial
+functions](https://trino.io/docs/current/functions/geospatial.html), and many
+others.
+
+From the core language perspective we added newer SQL functionality, such as
+[window functions and row pattern matching support]({% post_url
```

Comment:
```suggestion
[window functions and `MATCH_RECOGNIZE` support]({% post_url
```

Reply:
I've had people specifically call out this function as a reason they came to Trino. I think we should mention the SQL keyword. WDYT?